src/activities/act-7-2-2.xml

```diff
@@ -128,9 +128,48 @@
       <ol label="a.">
         <li>
           <p>
-            Answers for each of the prompts above.
+            When <m>y \lt  0</m> and when <m>y \gt 4</m>, <m>y</m> is a decreasing function of <m>t</m>.  When <m>0 \lt  y \lt  4</m>, <m>y</m> is a increasing function of <m>t</m>.
           </p>
+          <figure xml:id="F-act-7-2-2-a">
+            <caption>Plot of <m>\frac{dy}{dt}</m> versus <m>y</m> for <m>\frac{dy}{dt} = -\frac{1}{2}y(y-4)</m>.</caption>
+              <image source="images/7_2_Act2_dydt_plot_soln">
+              </image>
+          </figure>          
         </li>
+        <li>
+          <p>
+            <m>y = 0</m> and <m>y = 4</m>.
+          </p>
+        </li>
+        <li>
+          <figure xml:id="F-act-7-2-2-b">
+            <caption>Plot of the slope field for <m>\frac{dy}{dt} = -\frac{1}{2}(y - 4)</m>.</caption>
+              <image source="images/7_2_Act2_slope_field_soln">
+              </image>
+          </figure>           
+        </li>
+        <li>
+          <figure xml:id="F-act-7-2-2-c">
+            <caption>Plot of trajectories for <m>\frac{dy}{dt} = -\frac{1}{2}(y - 4)</m>.</caption>
+              <image source="images/7_2_Act2_trajectories_soln">
+              </image>
+          </figure>           
+        </li>
+        <li>
+          <p>
+            <m>y = 4</m> is stable; <m>y = 0</m> is unstable.
+          </p>
+        </li>                  
+        <li>
+          <p>
+            Tend to 4.
+          </p>
+        </li>
+        <li>
+          <p>
+            <xref ref="F-7-2-act-2-3a">Figure</xref> is for an ustable equilibrium.  <xref ref="F-7-2-act-2-3b">Figure</xref> is for a stable equilibrium.  
```

Comment:
Since you'd no longer have `@xml:id` for these things to refer back to, the cleanest thing is to put `@xml:id` on the `li` to which you want to refer. Then you can do something like "The image in `<xref ref="part-7-2-act-2-3a">part</xref>` is for…".

Reply:
I've fixed all of these figure references except this one and one related one (b instead of a), since "<figure xml:id="F-7-2-act-2-3a">" refers to a figure in the actual activity, not in a solution.

Reply:
Looks like my quoted text didn't show: figure xml:id="F-7-2-act-2-3a"